gen/calc.py

```diff
@@ -86,6 +86,22 @@ def calculate_mesos_slave_modules_json(mesos_slave_modules):
     return json_multiline.replace('\n', '\n' + injected_indent)
 
 
+def calculate_mesos_slave_modules(dcos_overlay_enable):
```

Comment:
Had a chat with @karya0. Looks like he should be adding the `--modules_dir` flag shortly. Once its merged upstream, I will go ahead and restructure this PR to use the `--modules_dir` flag on the Master and the Agent.